std_msgs/Header header
FootState[] states